docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Recommended Labels and Annotations
+---
+
+{% capture overview %}
+Many tools can be used to visualize and manage Kubernetes objects. The tools used
+to manage objects goes beyond kubectl and the dashboard. To enable interoperability
+between tools, a common set of labels and annotation allows objects to be described
+in a common manner that all tools can understand.
+
+In addition to supporting tooling, the recommended labels and annotations describe
```

Comment:
At some point in the document it would be worth digging into a couple concrete use cases that highlight the value of each of the proposed labels and annotations.